clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -1756,6 +1757,20 @@ ParseResult cir::FuncOp::parse(OpAsmParser &parser, OperationState &state) {
     return success();
   };
 
+  // Parse CXXSpecialMember attribute
+  if (parser.parseOptionalKeyword("special_member").succeeded()) {
```

Comment:
Can you add tests in `clang/test/CIR/IR/func.cir` to verify the parsing and printing of these attributes?